pandas/core/frame.py

```diff
@@ -5661,6 +5668,9 @@ def reset_index(
         col_fill : object, default ''
             If the columns have multiple levels, determines how the other
             levels are named. If None then the index name is repeated.
+        allow_duplicates : bool, default False
+            Allow duplicate column labels to be created.
+            Does not work if self.flags.allows_duplicate_labels is False.
```

Comment:
need a versionadded 1.4.0

Reply:
this statement doesn't make sense, instead put this in the Raises section

Reply:
Done